drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5276,7 +5275,7 @@ bool RasterizerStorageGLES2::free(RID p_rid) {
 
 	if (render_target_owner.owns(p_rid)) {
 
-		RenderTarget *rt = render_target_owner.getornull(p_rid);
+		RenderTarget *rt = render_target_owner.get(p_rid);
```

Comment:
https://github.com/godotengine/godot/commit/05bbb23b6a5368f4c4ee9211d2b92dbd511db6d4#diff-a61d01bcaef4874b6713c0b00e6d4571

* Removed redudant methods get and getptr, only getornull is supported now.

Seems that this is going to be changed to be the other way around in the future.